paper/sections/2preliminaries.tex

```diff
@@ -5,7 +5,7 @@ \subsection{PoW-based consensus and mining}
 Miners participate in PoW-based consensus by \emph{mining}, which works as follows.
 % how mining works
 Given the blockchain's latest view, the miner creates a block template $t$, which consists of transactions to include, hash of the last block, and other metadata.
-The miner then tries to solve a PoW puzzle that, the miner should find a nonce $n$ such that $H(t || n)$ is smaller than a difficulty parameter $T$.
+The miner then tries to find a nonce $n$ to solve a PoW puzzle, so that $H(t || n)$ is smaller than a difficulty parameter $T$.
```

Comment:
I prefer the previous version. In the new version, PoW puzzle is defined nowhere (`so that` follows the previous sentence), but in fact the PoW puzzle is the entire process that `given t, find n so that ...`